### PR TITLE
Add csv script, kdenlive marker script, kml map generator and add HERO5 support.

### PR DESCRIPTION
**Pull request by KonradIT**
- Add csv script
- Add kdenlive marker script
- Add kml map generator
- add HERO5 support

### CODE REVIEW

**Comment by stilldavid** [comment]
Yo! First, apologies for neglecting this PR so long. Second, thanks and great work!

This implements a lot of new things that are better suited to individual branches and pull requests. I'll check out and review the changes in more detail, but it's hard to reason about and potentially discuss lots of things changing in one PR. I'll try to add comments and get this merged, but in the future new features should be individual PRs.

I do have some concerns about some formatting as well. At the very least, you should run `go fmt` and `go vet` on the changes before they can be merged.

**Comment by KonradIT** [comment]
Done :)

**Comment by KonradIT** [comment]
Done :)

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ GoPro Metadata Format Parser
 
 TLDR:
 
-1. `ffmpeg -y -i GOPR0001.MP4 -codec copy -map 0:m:handler_name:"	GoPro MET" -f rawvideo GOPR0001.bin`
-2. `gopro2json -i GOPR0001.bin -o GOPR0001.json`
-3. There is no step 3
+1. `ffmpeg -y -i GOPR0001.MP4 -codec copy -map 0:3 -f rawvideo GOPR0001.bin`
```
Comment (JuanIrache) [inline README.md]: The data is not always in 0:3 (on firmware v2.00 it is not)
Instead, this always gets it, but in the .md file the tab character before GoPro is often displayed as a space.
0:m:handler_name:"	GoPro MET" 
In my repo I solved it by separating this line from the rest: https://github.com/JuanIrache/gopro-utils

Comment (stilldavid) [inline README.md]: Exactly, and I merged that change :) I had a hard time getting the tab character to work and be copy/paste friendly. Silly how GoPro put a tab character in the handler name!

Comment (stilldavid) [inline README.md]: Nice! I had a hard time getting the tab to work. I'll update the repo to put that on its own line.

**Comment by KonradIT** [comment]
Added HERO6 extraction script

**Comment by legrego** [comment]
Aside from the merge conflicts, is there any update on getting this PR included?
